analyzers/its/regression-test.ps1

```diff
@@ -28,13 +28,40 @@ if ($PSBoundParameters['Verbose'] -Or $PSBoundParameters['Debug']) {
     $global:DebugPreference = "Continue"
 }
 
+function Prepare-Project([string]$ProjectName){
+    $Output = ".\output\$ProjectName"
+    New-Item -ItemType directory -Path $Output | out-null
+
+    $SourcePath = ".\config\$ProjectName\SonarLint.xml"
+    if(-Not (Test-Path $SourcePath)){
+        $SourcePath = ".\config\SonarLint.xml"
+    }
+    $Content = Get-Content -Path $SourcePath -Raw
+
+    if($ruleId){
+        $RuleFragment = "    <Rule><Key>$ruleId</Key></Rule>"
+    } else {
```

Comment:
This logic is new.
_Old behavior:_ When $ruleId input parameter was specified, analyzer run this rule and all security hotspots. Creating some noise and wait time on the way.
_New behavior:_ Run only $ruleId